### DISCUSSION TITLE
Can I print UTF-8 characters?

### DISCUSSION THREAD

**Discussion by ViniciusHSAraujo** [kakzaki/blue_thermal_printer#160] [General]
I saw that there is the charset configuration, but where do I find which ones are compatible? Needed to print utf8 characters, such as "ãáàõ"

**Comment by ViniciusHSAraujo** [comment]
Example: Vinícius Araújo
![MVIMG_20210218_122614.jpg](https://user-images.githubusercontent.com/44041869/108379260-84d58500-71e4-11eb-8786-8312979a5d7c.jpg)

**Comment by TiagoPaza** [comment]
Use `charset: "iso-8859-1"` in the printCustom, to make accentuation.

**Comment by ebrahimmansur** [comment]
is there for arabic

**Comment by kreemov** [comment]
> is there for arabic

windows-1256 worked with me

**Comment by clooes** [comment]
`charset: "gbk"`

**Comment by ebrahimmansur** [comment]
> > is there for arabic
> 
> windows-1256 worked with me

how? ...show me gist of code

**Comment by ABDERRAOUFMERAZKA** [comment]
any update for this issues please ?
i can't display french accented letters like : 'téléphone'

**Comment by bluemix** [comment]
I've used this method:
https://github.com/kakzaki/blue_thermal_printer/issues/82#issuecomment-1048964685

**Comment by mviniciuscorrea** [comment]
> I saw that there is the charset configuration, but where do I find which ones are compatible? Needed to print utf8 characters, such as "ãáàõ"

same problem here... any solution? UTF8, iso-8859-1 and windows-1256 not work for me

**Comment by blitzlepe** [comment]
> > I saw that there is the charset configuration, but where do I find which ones are compatible? Needed to print utf8 characters, such as "ãáàõ"
> 
> same problem here... any solution? UTF8, iso-8859-1 and windows-1256 not work for me

Same problem, tried all charset above:
```
String txt = "Juárez";
bluetooth.printCustom(txt, Size.bold.val, Align.left.val, charset:" iso-8859-1")
```
á is not printed at all or a rare symbol
